### PR TITLE
a few more operators

### PR DESCRIPTION
**Pull request by matthiasdiener**

### CODE REVIEW

File: pytato/array.py [hunk 1]
```diff
@@ -537,6 +544,16 @@ def _unary_op(self, op: Any) -> Array:
...
 
+    __and__ = partialmethod(_binary_op, operator.and_)
+    __rand__ = partialmethod(_binary_op, operator.and_, reverse=True)
+    __or__ = partialmethod(_binary_op, operator.or_)
+    __ror__ = partialmethod(_binary_op, operator.or_, reverse=True)
+    __xor__ = partialmethod(_binary_op, operator.xor)
+    __rxor__ = partialmethod(_binary_op, operator.xor, reverse=True)
+
+    def __abs__(self) -> Array:
+        return abs(self)  # type: ignore
```
Comment (inducer) [inline pytato/array.py:555]: Won't this lead to infinite recursion?

File: pytato/array.py [hunk 2]
```diff
@@ -537,6 +544,16 @@ def _unary_op(self, op: Any) -> Array:
...
+
```
Comment (inducer) [inline pytato/array.py:556]: Tests?

**Review by inducer — COMMENTED** [review]
Thanks!

File: pytato/array.py (hunk 1 above)
Comment (kaushikcfd) [reply pytato/array.py:555]: I think this would pick up https://documen.tician.de/pytato/array.html?highlight=abs#pytato.abs.

**Review by kaushikcfd — APPROVED** [review]
Except the tests pointed out by inducer, LGTM from my side as well. Thank you!

File: pytato/array.py (hunk 1 above)
Comment (matthiasdiener) [reply pytato/array.py:555]: Yeah, it worked fine in my tests.

File: pytato/array.py (hunk 2 above)
Comment (matthiasdiener) [reply pytato/array.py:556]: Added in https://github.com/inducer/pytato/pull/86/commits/e96cfc101c0b06ecef2508782ccbb0594ea3ed77

File: pytato/array.py (hunk 1 above)
Comment (inducer) [reply pytato/array.py]: Added a comment to un-confuse future self in bb78065. Also, used a different solution to the typing problem.

**Comment by inducer** [comment]
Thanks!

**Pull request merged**